internal/awsservice/dynamodb.go

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Similar to the [transmitter](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/performancetest/transmitter.go). However, only performing [updating item](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/performancetest/transmitter.go#L188-L254) and [query the item](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/performancetest/transmitter.go#L275-L296)